src/Appwrite/Query.cs

```diff
@@ -63,17 +63,17 @@ public static string EndsWith(string attribute, string value)
 
         public static string Between(string attribute, string start, string end)
         {
-            return AddQuery(attribute, "between", new List<string> { start, end });
+            return $"between(\"{attribute}\", \"{start}\", \"{end}\")";
         }
 
         public static string Between(string attribute, int start, int end)
         {
-            return AddQuery(attribute, "between", new List<int> { start, end });
+            return $"between(\"{attribute}\", {start}, {end})";
         }
 
         public static string Between(string attribute, double start, double end)
         {
-            return AddQuery(attribute, "between", new List<double> { start, end });
+            return $"between(\"{attribute}\", {start}, {end})";
```

Comment:
I think we should do `ParseValues(start)` and `ParseValues(end)` in all 3 scenarios

Reply:
Addressed in generator for next release :+1: